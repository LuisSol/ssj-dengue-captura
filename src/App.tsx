import React from "react";
import { ThemeProvider } from "styled-components";
import { ToastContainer } from "react-toastify";

import "react-toastify/dist/ReactToastify.css";

// styles
import { nomalTheme } from "./styles/theme";

// components
import AppRouter from "./routers/AppRouter";

// Context
import { MasterFormProvider } from "./Contexts/MainForm";

function App() {
  return (
    <ThemeProvider theme={nomalTheme}>
      <MasterFormProvider>
        <AppRouter />
      </MasterFormProvider>
      <ToastContainer />
    </ThemeProvider>
  );
}

export default App;
