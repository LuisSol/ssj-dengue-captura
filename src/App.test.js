import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

test("renders app without crashing", () => {
  global.matchMedia = jest.fn().mockImplementation(() => ({
    addEventListener: jest.fn(),
    removeEventListener: jest.fn(),
    matches: true,
  }));

  const { container } = render(<App />);

  expect(container).toMatchSnapshot();
});
