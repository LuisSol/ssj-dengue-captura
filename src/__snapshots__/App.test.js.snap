// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app without crashing 1`] = `
<div>
  <div>
    Hello
  </div>
</div>
`;
